crates/oxc_query/src/adapter.rs

```diff
@@ -23,7 +23,7 @@ pub struct Adapter<'a> {
     pub path_components: Vec<Option<String>>,
 }
 
-const SCHEMA_TEXT: &str = include_str!("./schema.graphql");
+pub const SCHEMA_TEXT: &str = include_str!("./schema.graphql");
```

Comment:
Is it a bad idea to expose our schema text as part of the crate? I don't want to give any guarantees on it...

Reply:
This is an internal crate so is fine I think.

Reply:
Sounds good